### ISSUE TITLE
[Screen Reader – Make Code - Explorer] Screen reader announces incorrect state as 'Selected/Non-selected' at 'Explorer' button.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3648]
**User Experience:**
Screen reader user won't get correct state of any control if screen reader provides incorrect state for that control and also he/she might get confused for such control. Not only this they won't be able to proceed further because of such confusion and also won't be able to proceed further. 

**Note:** User credentials should NOT be included in the bug. 

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Java Script" drop down and select it.
6. Navigate to "Python" option and select it.
7. Navigate to "Explorer" and select it.
8. Navigate through all the controls after expanding Explorer control.
9. Verify that state of 'Explorer' button by screen reader.

**Actual Result:**
Screen reader provides incorrect state as 'Selected/Non-selected' at 'Explorer' button. It is reading as 'Collapse, non-selected & Expanded, Selected'.

**Expected Result:**
Screen reader should provide correct state as 'Collapse/Expanded' at 'Explorer' button. It should be read as 'Explorer button collapse & explorer button expanded'.

**Note:**
Same issue exists on Firefox + NVDA & Chrome + JAWS.

**WCAG Reference:**
[WCAG 1.3.1 - Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG1.3.1; #Win10-Edge(Chromium)

[WCAG1.3.1_Explorer_Incorrect state.zip](https://github.com/microsoft/pxt-microbit/files/5522624/WCAG1.3.1_Explorer_Incorrect.state.zip)

**Comment by jwunderl** [comment]
no repro
